src/components/project/ProjectPreview.js

```diff
@@ -1,87 +1,56 @@
 import React from "react";
-import styled from "styled-components";
-import Button from "../general/Button";
+import {
+  Typography,
+  Card,
+  CardActionArea,
+  CardMedia,
+  CardContent,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText
+} from "@material-ui/core";
+import PlaceIcon from "@material-ui/icons/Place";
+import BarChartIcon from "@material-ui/icons/BarChart";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+  media: {
+    height: 140
+  }
```

Comment:
I think displaying the image to the left over the whole height and with a limited width is a better approach, because it uses the space of the card more efficiently. Also by just showing a slim portion of the image over the whole width, the most important part of the image might be cut out. So I suggest putting the image to the left over the whole height and the text to the right.